cmd/query/main.go

```diff
@@ -132,7 +141,8 @@ func main() {
 	)
 
 	if error := command.Execute(); error != nil {
-		logger.Fatal(error.Error())
+		fmt.Println(error.Error())
```

Comment:
this is unfortunate, because a logger may print the error quite differently, e.g. with a stack trace, write to file, report to Sentry, etc. If we guaranteed that the command only returned errors when it can't init the logger, then it wouldn't be too bad, but I don't think this is generally the case.

Reply:
This statement is invoked when: 
* an error is propagated from `runE` -> in all cases except logger creation and config file parsing `zap.Fatal` is called which directly calls `os.exit`.
* cobra fails to parse flags e.g. a wrong/undefined flag

Other solution might be to use a default logger here.